tests/test_data.py

```diff
@@ -14,22 +14,40 @@
 DEFAULT_CONTEXT_PATH = os.path.join(schema_dir, "sssom.context.jsonld")
 
 
-def get_test_file(filename):
+def get_test_file(filename: TextIO) -> str:
+    """Get test file.
+
+    :param filename: Filename
```

Comment:
Is TextIO really the right type annotation? Looks like it should be a str or Path

Reply:
Fixed in b47274d